exonum/src/api/node/public/explorer.rs

```diff
@@ -164,6 +164,11 @@ impl ExplorerApi {
         }
 
         let (upper, upper_bound) = if let Some(upper) = query.latest {
+            if upper > explorer.height() {
+                return Err(ApiError::NotFound(
+                    "Requested latest height is not found".to_string(),
```

Comment:
I'd include the actual parameter and its bound:

"Requested latest height (%s) is greater than the current blockchain height (%s)"